### PR TITLE
[Ready for Review] Added some stylings to user registration form and Updated Bootstrap to work properly

### LINKED ISSUE #36: Bootstrap Errors and no Mobile Responsiveness
As @picklu pointed out, using just Bootstrap, in the way in which it was implemented, was problematic. In fact, we had no mobile-responsiveness as it was. 
I added the popper.js and jQuery  <script>s and added a toggler icon and dropdown capability to the navbar making it mobile responsive.
I used the Bootstrap stylesheet link from getBootstrap.com and the "Uncaught TypeError: Cannot read property 'fn' of undefined" error disappeared.
So now the the stylesheet link and three js Bootstrap <script>s come from the same source, and things are working nicely with no errors.

Base code will reflect this changes as soon as PR #35 is approved.

### PR DESCRIPTION
**Pull request by willjw3**
fixes #36 and #37

### CODE REVIEW

**Review by allanapplebee — APPROVED** [review]
@willjw3 looks good to me. I'm happy to go ahead and merge this

File: client/pages/signup.jsx [hunk 1]
```diff
@@ -8,58 +8,97 @@ export const addUserMutation = gql`
...
+                  setSuccessMessage("Your account has been created!");
+                }
                 setEmail("");
                 setPassword("");
               }}
             >
               Sign in
             </button>
           </form>
+          <p className="text-center text-danger">{errorMessage}</p>
```
Comment (hitochan777) [inline client/pages/signup.jsx]: I think you should not render this tag when `errorMessage` is empty.
The same goes for the line below.

**Review by hitochan777 — CHANGES_REQUESTED** [review]
I left a comment regarding error message, but other than that LGTM.

File: client/pages/signup.jsx (hunk 1 above)
Comment (willjw3) [reply client/pages/signup.jsx]: I'll see about modifying this.

Comment (willjw3) [reply client/pages/signup.jsx]: Changed to code to conditionally render the `<p>` tag with a message in the following way:
```
          {errorMessage.length ? (
              <p className="text-center text-danger">{errorMessage}</p>
          ) : ( successMessage.length > 0 &&
              <p className="text-center text-success">{successMessage}</p>
          )}
```

Comment (willjw3) [reply client/pages/signup.jsx]: Thanks, @allanapplebee  and @hitochan777 .

**Review by hitochan777 — APPROVED** [review]

**Pull request merged**